playbooks/install-apt-packages.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Install packages with apt
+  hosts: all
+  become: yes
+
+  vars:
+    - packages: # Packages to be installed
+      - vim
```

Comment:
what if I need to specify a version of that package to be installed

Reply:
Nice feature request. I will take that into a consideration.

Reply:
Introduced the requested feature, please find the commit below ([681d51d](https://github.com/TechProber/cloud-estate/pull/88/commits/681d51d4814019a8128e496c3cb40614df3b89e0))

Reply:
cool